### PR TITLE
Swaps non-mounted recharging guns to batteries

### PR DESCRIPTION
**Pull request by Anewbe**
Should recharge at roughly the same rate as currently.
Long-Term Goal: Get various slowly-recharging cells for RnD to print.

- [x] Still need to double check some things
- [x] Still need to finish discussion

### CODE REVIEW

**Comment by Belsima** [comment]
So, recharge-time is just gone?

**Comment by Anewbe** [comment]
No?

**Comment by Belsima** [comment]
Ah- my bad. Misread the code.

File: code/modules/power/cells/power_cells.dm [hunk 1]
```diff
@@ -1,137 +1,92 @@
...
+	charge = 0
+
+/obj/item/weapon/cell/infinite
+	name = "infinite-capacity power cell!"
+	icon_state = "icell"
+	origin_tech =  null
+	maxcharge = 30000 //determines how badly mobs get shocked
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 80)
+
+	check_charge()
```
Comment (Mechoid) [inline code/modules/power/cells/power_cells.dm]: Not swapped to absolute pathing?

Comment (Anewbe) [reply code/modules/power/cells/power_cells.dm]: Ah, good catch, that was a straight copy-paste from the old file.

**Pull request merged**